andela/mrm_front
# #156719254 Implement navigation and menu bars

#### Description

We implemented the navigation and menu bars on the design provided

#### What does this PR do 

Design Top Navbar and Menu based on the design provided

#### How can this be tested offline

Ensure you have the project setup locally then navigate to `http://mrm-dev.andela.com:8080/`

#### Relevant pivotal tracker story

#156719254

#### Screenshot

<img width="1440" alt="screen shot 2018-05-15 at 11 28 37" src="https://user-images.githubusercontent.com/19430799/40045593-6545e504-5833-11e8-852c-f1a9c1cdfcbd.png">
